### PR TITLE
SamyGo.RemoteLANControl version 0.8

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98663)

### CODE REVIEW

File: manifests/s/SamyGo/RemoteLANControl/0.8/SamyGo.RemoteLANControl.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: SamyGo.RemoteLANControl
+PackageVersion: 0.8
+Installers:
+- InstallerUrl: http://download.samygo.tv/PC%20Applications/Remote%20LAN%20Control%20(v0.8).zip
+  Architecture: neutral
+  InstallerType: zip
+  NestedInstallerType: exe
```
Comment (Trenly) [inline manifests/s/SamyGo/RemoteLANControl/0.8/SamyGo.RemoteLANControl.installer.yaml:10]:
```suggestion
  NestedInstallerType: portable
```

File: manifests/s/SamyGo/RemoteLANControl/0.8/SamyGo.RemoteLANControl.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SamyGo.RemoteLANControl
+PackageVersion: 0.8
+PackageLocale: en-US
+Publisher: SamyGo
+PackageName: Remote LAN Control
+License: http://download.samygo.tv/PC%20Applications/Remote%20LAN%20Control%20(v0.8).zip
```
Comment (Trenly) [inline manifests/s/SamyGo/RemoteLANControl/0.8/SamyGo.RemoteLANControl.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

File: manifests/s/SamyGo/RemoteLANControl/0.8/SamyGo.RemoteLANControl.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SamyGo.RemoteLANControl
+PackageVersion: 0.8
+PackageLocale: en-US
+Publisher: SamyGo
+PackageName: Remote LAN Control
+License: http://download.samygo.tv/PC%20Applications/Remote%20LAN%20Control%20(v0.8).zip
+ShortDescription: SamyGO Remote LAN
```
Comment (Trenly) [inline manifests/s/SamyGo/RemoteLANControl/0.8/SamyGo.RemoteLANControl.locale.en-US.yaml:10]: Please provide a better short description of what the package does

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — APPROVED** [review]

**Comment by upintheairsheep** [comment]
merge?

**Pull request merged**